aman/src/pages/Forgotpsswrd.jsx

```diff
@@ -0,0 +1,82 @@
+import { z } from "zod";
+import { useState } from "react";
+import Input from "../components/Input";
+const Forgotpsswrd = ({ handleClick }) => {
+  const [formData, setFormData] = useState({ email: "" });
+  const [error, setError] = useState({});
+  const schema = z.object({
+    email: z.string().email("Must be a valid email address"),
+  });
+
+  const handleChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const result = schema.safeParse({
+      ...formData,
+    });
+    if (!result.success) {
+      const formattedErrors = result.error.format();
+      setError(formattedErrors);
+    } else {
+      setError({});
+    }
+  };
+  return (
+    <div>
+      <div className="flex flex-col items-center justify-center m-4 mt-10">
+        <div className="rounded-lg shadow-lg  w-full max-w-md h-[510px] m-8 p-3  bg-white mb-18">
+          <div className="flex flex-col p-6 ">
+            <h3 className="font-bold text-xl text-center tracking-tight">
```

Comment:
Here, if you are using h3 tag and then defining the font weight and font height both then the use of h3 tag here is null. Instead then you can use a p tag. Also check for this at other places as well.